proarc-common/src/main/java/cz/cas/lib/proarc/common/catalog/MetadataItem.java

```diff
@@ -34,8 +34,15 @@
     private String preview;
     /** short descriptor of the item; used in ListGrid */
     private String title;
+    /** optional id from rd */
+    private Long rdczId;
 
-    public MetadataItem() {
+    public MetadataItem(int id,  Long rdczId, String mods, String preview, String title) {
```

Comment:
Původní konstruktor `MetadataItem()` byl nevyužitý?

Reply:
Přesně tak, nebyl